ansible/roles/base/vars/RedHat_7.yml

```diff
@@ -28,3 +28,21 @@ package_list:
   - compat-sap-c++-7
   - compat-sap-c++-9
```

Comment:
Since this role is common for all stacks and includes packages required for all roles I see some duplicate entries. We should put a check/logic based on the stack installed so we can exclude the ones not required for deploying a particular stack. I will do that as a separate PR with the testing completed on all stacks. 

Having a reference for list of packages required for a stack is good in my opinion so we don't have to track whether an SAP image has the packages pre-baked and ansible will skip the installation if done already.